.storybook/preview.tsx

```diff
@@ -11,6 +11,7 @@ import {
 import type { Preview, ReactRenderer } from '@storybook/react';
 
 import '../styles/new/index.css';
+import { NotificationProvider } from '../providers/notificationProvider';
```

Comment:
This import should be together with the rest of provider imports, 🙇

Reply:
updated